main.go

```diff
@@ -53,7 +49,6 @@ func init() {
 	flag.StringVar(&instanceOverride, "instance", "", "instance: Spanner instance to use")
 	flag.StringVar(&filePrefix, "prefix", "", "prefix: file prefix for generated files")
 	flag.StringVar(&driverName, "driver", "pg_dump", "driver name: flag for accessing source DB or dump files (accepted values are \"pg_dump\", \"postgres\", \"mysqldump\", and \"mysql\")")
-	flag.Int64Var(&schemaSampleSize, "schema-sample-size", int64(100000), "schema-sample-size: the number of rows to use for inferring schema (only for DynamoDB)")
```

Comment:
For the moment, suggest we leave schema-sample-size in this file, and just pass schemaSampleSize as a parameter to cmd.CommandLIne. Yeah, it adds an extra parameter, but there's already a lot of parameters in that function and at least its consistent and it also keeps all flags together.

I like your suggestion of collecting all of the config state in a struct. Basically grab all of the components we pass to CommandLine like: driverName, project, instance, dbName, dataOnly, schemaOnly, sessionJSON, filePrefix and schemaSampleSize. In other words all of the stuff we collect or compute from flags and environment variables.

But let's add a TODO for that and defer for a following PR.
Maybe we could even go a step further, and move code for computing these config values in main.go (currently 
GetProject, GetInstance and some related function are in conversion.go, however they aren't used by the webapi). That way we could have a really clean interface from main: main either calls web.WebAPI() or it calls cmd.CommandLine. That would be nice.

Reply:
I agree that we need to take a hard look at the content of conversion.go since right now it has become a dump all module which contains many unrelated functions. I have added a TODO for deferring the same for a future PR. Will also look into moving GetProject, GetInstance and related functions in that PR. Done @https://github.com/cloudspannerecosystem/harbourbridge/pull/121/commits/96fafc8df69686a5aeac12ee5bec2c5656666d1e.